website/docs/r/policy_set_definition.html.markdown

```diff
@@ -74,7 +76,11 @@ A `policy_definition_reference` block supports the following:
 
 * `policy_definition_id` - (Required) The ID of the policy definition or policy set definition that will be included in this policy set definition.
 
-* `parameters` - (Required) A mapping of the parameter values for the referenced policy rule. The keys are the parameter names.
+* `parameters` - (Optional / **Deprecated in favour of `parameter_values`**) A mapping of the parameter values for the referenced policy rule. The keys are the parameter names.
+
+-> **NOTE:** Since Terraform's concept of a map requires all of the elements to be of the same type, you could only assign string value as the parameter values when you are using the `parameters` attribute. Please use `parameter_values` instead.
```

Comment:
we should be able to remove this since we're pushing people over to the new field

Reply:
Removed